app/iot/service.py

```diff
@@ -1,8 +1,9 @@
+import asyncio
 import random
 import string
 from typing import Protocol
 
-from .message import Message, MessageType
+from app.iot.message import Message, MessageType
```

Comment:
Ensure that the 'Message' class from 'app.iot.message' has 'device_id', 'msg_type', and 'data' attributes, as they are used in the 'send_msg' method.